web/apps/labelstudio/src/components/Button/Button.jsx

```diff
@@ -5,7 +5,7 @@ import { FormSubmissionContext } from "../Form/FormContext";
 import "./Button.scss";
 
 export const Button = React.forwardRef(
-  ({ children, type, extra, className, size, waiting, icon, tag, look, ...rest }, ref) => {
+  ({ children, type, extra, className, rawClassName, size, waiting, icon, tag, look, ...rest }, ref) => {
```

Comment:
funnily enough this is implied to be in `rest` as it was being passed down to block regardlress so i've used it as such before - but im glad you made it more explicit